src/NUnitFramework/framework/Internal/TestParameters.cs

```diff
@@ -13,12 +13,7 @@ namespace NUnit.Framework.Internal
     /// </summary>
     public abstract class TestParameters : ITestData, IApplyToTest
     {
-        internal static readonly object[] NoArguments =
-#if NET45 // New in net46
```

Comment:
This crossed my radar because of the macro here, but I'll create a separate ticket to track a comprehensive adoption of Array.Empty<T>